api/base/serializers.py

```diff
@@ -29,6 +29,65 @@ def _url_val(val, obj, serializer, **kwargs):
         return val
 
 
+class HyperLinkedIdentityFieldWithMeta(ser.HyperlinkedIdentityField):
+
+    def __init__(self, view_name=None, **kwargs):
+        assert view_name is not None, 'The `view_name` argument is required.'
+        kwargs['read_only'] = True
+        kwargs['source'] = '*'
+        self.count = kwargs.pop('count', None)
```

Comment:
Changed so you can add a meta dictionary as an argument to HyperlinkedIdentityFieldWithMeta.  In the case of getting a node's children, the argument would be `meta = {'count': 'get_node_count'}`